doc/telluric.rst

```diff
@@ -0,0 +1,185 @@
+=======
```

Comment:
Thanks for this!  I fixed the heading things.

They have to be the same length as the text.

Reply:
Thanks!